lighthouse-core/report/html/report-styles.css

```diff
@@ -117,6 +122,7 @@
   --score-title-line-height-big: 36px;
   --score-title-line-height: 26px;
   --section-padding: 40px;
+  --shape-margin: 7px 12px 0 4px;
```

Comment:
looks like some things might have been merged back in?

Reply:
removed